cypress/e2e/tests/migration/applicationinventory/assessment/import.test.ts

```diff
@@ -45,10 +45,6 @@ describe(["@tier2"], "Application import operations", () => {
         deleteAppImportsTableRows();
     });
 
-    after("Perform test data clean up", function () {
-        deleteApplicationTableRows();
```

Comment:
I understand that we shouldn't use this method, but this test suite creates ~10 applications that will be left undeleted if we don't clear them.

Reply:
Hi @abrugaro , the current spec file has two after() hooks, one at the top and the other at the end of the file. I'm just removing the one at the top .